README.md

```
@@ -1,34 +1,67 @@
-
-If you find an issue or you want to propose an improvement, please feel free to post it on: [issues](https://github.com/rainlanguage/dotrain/issues)
+The Rain language server protocol
+([LSP](https://microsoft.github.io/language-server-protocol/)) services
+implementation (language services) and .rain composer written in rust and made
+available for NodeJs and broswers through
+[wasm-bindgen](https://rustwasm.github.io/docs/wasm-bindgen/) in
+Typescript/Javascript which makes it well suited for editors and IDEs (as it is
+used in Rainlang vscode and codemirror language extension).
+
+- Dotrain specs can be found
+  [here](https://github.com/rainlanguage/specs/blob/main/dotrain.md)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)